package.json

```diff
@@ -15,9 +15,9 @@
   },
   "dependencies": {
     "@ampproject/animations": "0.2.0",
+    "@ampproject/toolbox-cache-url": "2.2.0",
```

Comment:
Interesting to see this change here, did this automatically happen when you added `pumpify` as a devDependency?

Reply:
https://github.com/ampproject/amphtml/commit/ab0ec424843597f989e2a0b7d39797a1296d2444#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Reply:
Yeah, pretty sure `yarn` did this when `pumpify` was added.

Not sure why the original change wasn't alphabetized. @Enriqe, did you manually edit `package.json`?

Reply:
I might have 🤔

Reply:
I think it was just manually added.